library/core/src/ops/unsize.rs

```diff
@@ -78,7 +78,7 @@ impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<*const U> for *const T {}
 /// `m` defined as `fn m(&self);`. When calling `t.m()`, the receiver `t` is a wide pointer, but an
 /// implementation of `m` will expect a narrow pointer as `&self` (a reference to the concrete
 /// type). The compiler must generate an implicit conversion from the trait object/wide pointer to
-/// the concrete reference/narrow pointer. Implementing `DispatchFromDyn` indicates that that
```

Comment:
"that that" can be grammatically valid and I think it is in this context.

Reply:
Reverted, please check again.